src/app/src/pages/redteam/setup/components/Purpose.tsx

```diff
@@ -24,9 +24,19 @@ export default function Purpose({ onNext }: PromptsProps) {
   const { config, updateApplicationDefinition, setFullConfig } = useRedTeamConfig();
   const { recordEvent } = useTelemetry();
   const [confirmDialogOpen, setConfirmDialogOpen] = useState(false);
+  const [hasExternalSystems, setHasExternalSystems] = useState(false);
```

Comment:
Changes look good. I just am not sure if this should start open or closed. @sklein12 what do you think?